style.css

```diff
@@ -153,7 +153,7 @@
 
 @keyframes glowing {
   0% { box-shadow: 0 0 3px rgb(222, 190, 130) }
-  50% { box-shadow: 0 0 10px orange }
+  50% { box-shadow: 0 0 30px orange }
```

Comment:
This was definitely a little faint on the start screen. We should make it even more noticeable.